projects.json

```diff
@@ -834,6 +834,37 @@
       }
     ]
   },
+  {
+    "repository": "Git",
+    "url": "https://github.com/grpc/grpc-swift",
+    "path": "grpc-swift",
+    "branch": "main",
+    "maintainer": "gbarnett@apple.com",
+    "compatibility": [
+      {
+        "version": "5.2",
+        "commit": "a45db1b439bc85b9e18f7ee1a5bf3ec564107819"
+      },
+      {
+        "version": "5.3",
+        "commit": "a45db1b439bc85b9e18f7ee1a5bf3ec564107819"
+      }
+    ],
+    "platforms": [
+      "Darwin",
+      "Linux"
+    ],
+    "actions": [
+      {
+        "action": "BuildSwiftPackage",
+        "configuration": "release",
+        "tags": "sourcekit-disabled swiftpm"
```

Comment:
```suggestion
        "tags": "sourcekit-disabled swiftpm",
        "xfail": {
            "issue": "https://bugs.swift.org/browse/SR-14068",
            "compatibility": ["5.2", "5.3"],
            "branch": ["main"]
        }
```

Reply:
We can still merge this if we add an xfail here since we have a bug tracking the regression.